data/schema/account/account.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package schema;
+
+import "google/protobuf/timestamp.proto";
+
+// Account Schema Storage Types
+
+enum AccountType {
+  USER=0;
+  ORGANIZATION=1;
+}
+message Account {
+  string  id=1;
```

Comment:
Nitpicking here (but then again, Go itself is quite opinionated, so we should be used to that) :p

    string id = 1;

The [protobuf style guide](https://developers.google.com/protocol-buffers/docs/style) isn't explicit about it, but that's how it shows them, and that's consistent with all of our other proto files...